public/js/components/RightSidebar.js

```diff
@@ -144,24 +144,24 @@ const RightSidebar = React.createClass({
     const { shouldPauseOnExceptions, shouldIgnoreCaughtExceptions,
             pauseOnExceptions } = this.props;
 
-    function pauseBtn(pause, ignore, tooltip) {
+    function pauseBtn(pause, ignore, tooltip, className) {
       return debugBtn(
         () => pauseOnExceptions(pause, ignore),
-        "pause-exceptions",
+        className,
         "enabled",
         tooltip
       );
     }
 
     if (!shouldPauseOnExceptions && !shouldIgnoreCaughtExceptions) {
```

Comment:
I feel like this code is really duplicative for adding CSS values and a label. I think a better solution would be to remove the `pauseBtn` function and call `DebugBtn` directly and use the `classnames` helper to resolve the class to use. I'd also modify the classes to essentially be `.pause-btn.ignore`, `.pause-btn.uncaught` and `.pause-btn.all` so you can more easily use `classnames`.